lib/rucio/common/exception.py

```diff
@@ -811,3 +820,13 @@ def __init__(self, *args, **kwargs):
         super(MissingModuleException, self).__init__(args, kwargs)
         self._message = "The module is not installed."
         self.error_code = 75
+
+
+class ServerConnectionException(RucioException):
+    """
+    RucioException
+    """
+    def __init__(self, *args, **kwargs):
+        super(ServerConnectionException, self).__init__(args, kwargs)
+        self._message = "Cannot connect to the Rucio server."
+        self.error_code = 76
```

Comment:
NotAllFilesDownloaded already has errorcode 76 ;-)

and MissingModuleException just above has 75, which is already claimed by NoFilesDownloaded.

Can you please set:  MissingModuleException=77, and ServerConnectionException=78